### PR TITLE
Add support for multi-value parameters in API Gateway proxy events

### PR DESCRIPTION
**Pull request by piotrkubisa**
Recently I was digging more into Lambda environment and I found new fields which are not defined in `aws-lambda-go` struct for API Gateway Proxy Request event:

- `events.APIGatewayProxyRequest.MultiValueHeaders`.
- `events.APIGatewayProxyRequest.MultiValueQueryStringParameters`.
- `events.APIGatewayRequestIdentity.AccessKey` (populated whether doing an authorization using AWS credentials).

https://aws.amazon.com/about-aws/whats-new/2018/10/amazon-api-gateway-adds-support-for-multi-parameters/

Maintainers: do we need any extra unit test for these new fields?

### CODE REVIEW

**Comment by piotrkubisa** [comment]
Yesterday I found also a blog post about the multi-value parameters on AWS Compute Blog https://aws.amazon.com/blogs/compute/support-for-multi-value-parameters-in-amazon-api-gateway/ Is there anything what I can do in this PR to help you introduce this this change to `aws-lambda-go` repository?

**Comment by harrisonhjones** [comment]
I haven't dove too deeply into the testing of this package but it seems like you did change some test data json files. If you updated the tests that relied on those files to check for the new properties (`events.APIGatewayProxyRequest.MultiValueHeaders`, `events.APIGatewayProxyRequest.MultiValueQueryStringParameters`, & 
`events.APIGatewayRequestIdentity.AccessKey`) and their correct values that would be sufficient.

File: events/testdata/apigw-request.json [hunk 1]
```diff
@@ -42,7 +66,8 @@
 			"accountId": "theAccountId",
 			"cognitoIdentityId": "theCognitoIdentityId",
 			"caller": "theCaller",
-			"apiKey": "theApiKey",
+            "apiKey": "theApiKey",
+            "accessKey": "ANEXAMPLEOFACCESSKEY",
```
Comment (bmoffatt) [inline events/testdata/apigw-request.json]: thanks for adding this as well

**Comment by bmoffatt** [comment]
LGTM, thanks!

**Pull request merged**